### PR TITLE
Authentication through template!

### PR DESCRIPTION
**Pull request by TarunTomar122**

### CODE REVIEW

File: App/Containers/SplashScreen/SplashScreen.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { Text, View,Button } from 'react-native'
+import styles from './SplashScreenStyle'
+import { connect } from 'react-redux'
+import ExampleActions from 'App/Stores/Authentication/Actions'
+import { PropTypes } from 'prop-types'
+
+class SplashScreen extends React.Component {
+  render() {
+    this.props.fetchUser()
```
Comment (kunaltawatia) [inline App/Containers/SplashScreen/SplashScreen.js:10]: We never call a function which manage state from inside the render function, such actions can somewhere lead to infinite loops.

File: App/Containers/SplashScreen/SplashScreen.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { Text, View,Button } from 'react-native'
```
Comment (kunaltawatia) [inline App/Containers/SplashScreen/SplashScreen.js:2]: file formatting issues.

File: App/Containers/SplashScreen/SplashScreen.js (hunk 1 above)
Comment (TarunTomar122) [reply App/Containers/SplashScreen/SplashScreen.js]: I have Fixed this issue, Please review my pull request once again!

File: App/Containers/SplashScreen/SplashScreen.js (hunk 2 above)
Comment (TarunTomar122) [reply App/Containers/SplashScreen/SplashScreen.js]: I have formatted all the files now, so please review it once again!

**Comment by kunaltawatia** [comment]
> Great Work :100:

**Pull request merged**